apis/nucleus/src/components/listbox/ListBoxRowColumn.jsx

```diff
@@ -20,36 +28,62 @@ const useStyles = makeStyles((theme) => ({
   column: {
     flexWrap: 'nowrap',
     borderRight: `1px solid ${theme.palette.divider}`,
+    color: theme.palette.text.primary,
     '&:focus': {
       boxShadow: `inset 0 0 0 2px ${theme.palette.custom.focusOutline}`,
       outline: 'none',
     },
   },
+
+  // The interior wrapper for all field content.
   cell: {
-    overflow: 'hidden',
-    textOverflow: 'ellipsis',
     display: 'flex',
     alignItems: 'center',
     minWidth: 0,
     flexGrow: 1,
-    '& span': {
-      whiteSpace: 'nowrap',
-      fontSize: '12px',
-      lineHeight: '16px',
-      userSelect: 'none',
+    // Note that this padding is overridden when using checkboxes.
+    paddingLeft: '12px',
+    paddingRight: '12px',
+  },
+
+  // The leaf node, containing the label text.
+  labelText: {
+    flexBasis: 'max-content',
+    fontSize: '0.9rem',
```

Comment:
Any specific reason to why you are using rem here instead of px?
(At some point I guess we'd like to tie it to the theme as well...)

Reply:
Not really, but `rem` will automatically adapt to the root font-size automatically. Is there any reason we want `px` instead of `rem`?

Reply:
I think the general advice is to use adaptive font-sizes but we tend to only use `px` in our implementations. Do you know if we have any "best practices" for this at Qlik?

Reply:
I try to stick with the theme configured for Material UI. In this case there is a set a typography variants specified so I would pick a suitable one from the list and apply its styling. If the size, colors or other styling is desired to be altered the theme config can be updated.

In the storybook for sprout there is a list of available typography variants:
http://rd-sprout.qliktech.com/sprout/?path=/story/mui-components-typography--variants

Reply:
I've switched to using `variant="body2"` which seems to be closest to what I already used (`0.875rem` instead of `0.9rem`).